hivemind_core/service.py

```
@@ -121,14 +116,21 @@ def run(self):
 
-        # start network protocol that will deliver HiveMessages
-        network_class, net_config = get_network_protocol()
-        LOG.info(f"Network protocol: {network_class.__name__}")
+        # start network protocols that will carry HiveMessages
+        protos = []
+        for plug_name, plug_conf in get_server_config()["network_protocol"].items():
+            try:
+                network_class = NetworkProtocolFactory.get_class(plug_name)
+                LOG.info(f"Network protocol: {network_class.__name__}")
+                protos.append(network_class(hm_protocol=hm_protocol, config=plug_conf))
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)